migrations/versions/f49553f399f3_.py

```diff
@@ -0,0 +1,75 @@
+"""empty message
+
+Revision ID: f49553f399f3
+Revises: 
+Create Date: 2020-09-09 15:41:57.916553
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'f49553f399f3'
+down_revision = None
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
```

Comment:
we had to make some changes to column datatypes - no easy way to do it, so we just redid all the migrations